.github/workflows/build-dependabot.yml

```
@@ -30,10 +30,10 @@ jobs:
       - name: 🧹 Remove toolchains.xml
         run: rm $HOME/.m2/toolchains.xml || true
 
-      - name: 🛠️ Set up JDK 17
+      - name: 🛠️ Set up JDK 21
         uses: actions/setup-java@v3
```

actionlint (1.7.7)

the runner of "actions/setup-java@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)